src/schema/v2/users/deleteUserInterestForUserMutation.ts

```diff
@@ -0,0 +1,103 @@
+import { GraphQLObjectType, GraphQLString } from "graphql"
+import { mutationWithClientMutationId } from "graphql-relay"
+import { GraphQLNonNull } from "graphql"
+import { ResolverContext } from "types/graphql"
+import { UserInterest, userInterestType } from "../userInterests"
+import { snakeCase } from "lodash"
+import {
+  formatGravityError,
+  GravityMutationErrorType,
+} from "lib/gravityErrorHandler"
+import { GraphQLUnionType } from "graphql"
+
+interface Input {
+  id: string
+  anonymousSessionId?: string
```

Comment:
I'm not sure I understand these other inputs (although they are marked optional). Deleting a user interest (for a user), just needs the `id` of the user interest, no? (at least in the context of how we're using this in Forque).

Reply:
Another good point. I left those in thinking all endpoint params should be mapped out but I suppose for these mutations those arent strictly necessary and I cant think of a reason to keep em. I removed them in https://github.com/artsy/metaphysics/pull/4582/commits/dc2accb5bec8cc15b807c1316b1caa2fc94b5b2a, which also has some additional changes. Could you please look that over?